db/migrate/20190313140443_relax_fixed_asset_constraints_for_none_deprecation_method.rb

```diff
@@ -0,0 +1,15 @@
+class RelaxFixedAssetConstraintsForNoneDeprecationMethod < ActiveRecord::Migration
+  def change
+    reversible do |r|
+      r.up do
+        change_column :fixed_assets, :stopped_on, :date, null: true
```

Comment:
Shouldn't this be done for `fixed_asset_depreciations` too?

Reply:
Nope, because we don't generate any FixedAssetDepreciation when creating a FixedAsset that does not devaluate

Reply:
Then you cannot relax the validation in the `FixedAssetDepreciation` model, since FixedAssetDepreciation#stopped_on still has the DB constraint `NOT NULL`.